### PR TITLE
Update onboarding docs

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: site/content/internal/onboarding/manager-guide.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Manager Guide
+date: 2017-11-07T14:28:35-05:00
+subsection: Onboarding
+weight: 30
+---
+
+### Mentor For The Day
+- Schedule a 1-1 each day bewteen new staff and team members (including QA and PM) and other devs who would be relevant to mee with for the first two weeks
+- Meet with the new staff member for 5-10 minutes everyday for the first week or so to make sure everything is going well
```
Comment (coreyhulen) [inline site/content/internal/onboarding/manager-guide.md]: 5-10 mins or 20-30min ?  should also include something like "hangout and get to know each other"

Comment (jwilander) [reply site/content/internal/onboarding/manager-guide.md]: Meant for the manager to meet with the new staff member everyday in addition to the mentor meeting, not for the 1-1 between the mentor and new staff. I can update the text to be clearer about that

Comment (amyblais) [inline site/content/internal/onboarding/manager-guide.md]: everyday > every day

File: site/content/internal/onboarding/manager-guide.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Meetings and Accounts
+- Add new staff member to all appropriate meetings
+
+### OneLogin Account Set-up
+- Login to OneLogin and go to https://mattermost.onelogin.com/admin
+- Click Users -> All Users > New User
+- Fill out first/last name, username, manager (use email minus @mattermost.com as username)
+- Save user then go to Applications tab
+- Add them to Developer, VPN, Vault, Jenkins groups and save
```
Comment (amyblais) [inline site/content/internal/onboarding/manager-guide.md]: Add comma between "user" and "then"

Add "and" between Vault and Jenkins

File: site/content/internal/onboarding/manager-guide.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- For the new user, go to More Actions and click Send Invitation (without this they won't get their invite)
+- If the user is marked as unlicensed, need to ask Carlos or Corey to increase OneLogin seat count
```
Comment (amyblais) [inline site/content/internal/onboarding/manager-guide.md]: Remove "need to" (0/5)

**Pull request merged**